README.md

```diff
@@ -1,24 +1,180 @@
+
+  * Get started with [Ruby on Rails](https://guides.rubyonrails.org/getting_started.html).
+
+  * Make sure you have [PostgreSQL](https://www.postgresql.org/) installed and running.
+
+### Setup
+
+Clone this repository to your desired folder or download the Zip folder:
+
+```
+git clone https://github.com/ndikumanaisaie/blog_app_ror.git
+```
+
+  * Navigate to the location of the folder in your machine:
+
+  **`you@your-Pc-name:~$ cd blog_app_ror`**
+
+### Install
+
+To install all gems, run:
+
+```
+bundle install
+```
+
+### Usage
+
+To run the project, follow these instructions:
+
+  * After Cloning this repo to your local machine.
+  * To get it running in your terminal, run it in your IDE
+
+### Demo Video
+
+  * No video
+
+### Running the app
+
+  *  Run rake `db:create:all` and `rake db:migrate`
+  *  Run `rake db:seed` to populate the database with some sample data.
+  *  Run `rails s` to start the server.
+  *  In your browser, go to `http://localhost:3000`.
+
+### Run tests
+
+  * no tests
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👤 Authors <a name="authors"></a>
+
+👤 **Ndikumana Isaie**
+
+- GitHub: [ndikumanaisaie](https://github.com/ndikumanaisaie)
+- Twitter: [Ndikuma38670724](https://twitter.com/Ndikuma38670724)
+- LinkedIn: [Ndikumana Isaie](https://www.linkedin.com/in/ndikumanaisaie/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Implement more methods.**
+- [ ] **Write Extra Message.**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+Give a ⭐️ if you like this project and how I managed to build it!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- The original design ideal from Microverse💕.
+- Project from [Microverse](https://bit.ly/MicroverseTN) Ruby / Database and Interviewing module.
+- Thanks to the Microverse team for the great curriculum.
+- Thanks to the Code Reviewer(s) for the insightful feedbacks.
+- A great thanks to My coding partner(s), morning session team, and standup team for their contributions.
+- Hat tip to anyone whose code was used.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ❓ FAQ <a name="faq"></a>
+
+- **Can I fork and reuse the repository**
+
+  - Yes please, feel free.
+
+- **Can I improve the repository, and my changes will be accepted if they are good?**
+
+  - Yes please, nice ideas are welcome, please.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ Required ] Great job so far, I think it will be even much better if you add a `LICENSE` file in the root directory of your project. Currently, this link leads to a 404 page not found error which is not very professional. Please fix this, you can use the contents of the MIT License on this [Link >>](https://choosealicense.com/licenses/mit/).

![Screenshot from 2023-02-21 18-26-17](https://user-images.githubusercontent.com/16392046/220386977-a5a7cf32-d691-4a72-9055-863446a61b48.png)